Library/Homebrew/test/utils/shell_spec.rb

```diff
@@ -19,8 +19,15 @@
       expect(subject.profile).to eq("~/.bash_profile")
     end
 
+    it "returns /tmp/.zshrc for Zsh if ZDOTDIR is /tmp" do
+      ENV["SHELL"] = "/bin/zsh"
+      ENV["ZDOTDIR"] = "/tmp"
+      expect(subject.profile).to eq("/tmp/.zshrc")
+    end
+
     it "returns ~/.zshrc for Zsh" do
       ENV["SHELL"] = "/bin/zsh"
+      ENV["ZDOTDIR"] = nil
```

Comment:
```suggestion
```

Reply:
For some reason this test, without unsetting ZDOTDIR, worked on my machine where ZDOTDIR is set in zsh. I expected it to fail when run without edits.

Does `brew test` run the tests in an isolated bash shell or something?

Reply:
> Does `brew test` run the tests in an isolated bash shell or something?

Yes.